web-ui/src/main/resources/catalog/components/search/resultsview/partials/viewtemplates/grid.html

```diff
@@ -75,7 +75,12 @@ <h1>
              title="{{'workingCopy' | translate}}"
              class="gn-workingcopy-status pull-right">
           <i class="fa fa-pencil"></i>
-          <span>{{'workingCopy' | translate}}</span>
+          <a href=""
+             data-ng-show="md.isTemplate != 's'"
+             data-ng-href="catalog.search#/metadraf/{{md.getUuid()}}"
+             title="{{md.title || md.defaultTitle}}">
```

Comment:
Shouldn't the title of this link contain "Working copy" or "Edit working copy"? This would be more in-line with the other icons on the page. i.e. the other pencil has "Edit" when you hover over the icon.

Reply:
I don't think we should put "editor working copy" text. The intention is not amending the text but adding a hyper link. The text was there and it wasn't clickable. See issue https://github.com/geonetwork/core-geonetwork/issues/6198

Reply:
`title="{{'workingCopy' | translate}}"` already exists in the `<div`  I'm not sure why you would want the link to override the title to be the metadata title?

Reply:
I moved the pencil into the link. I also swapped the title for the default tittle which is also used on editor page.

![image](https://user-images.githubusercontent.com/74916635/173083771-88f18754-fef8-4032-8d1c-4d1764ea58f7.png)